cmd/rokka/main.go

```diff
@@ -46,5 +48,29 @@ func main() {
 		APIKey: cfg.APIKey,
 	})
 
-	cli.ExecCommand(cl, args)
+	err = cli.ExecCommand(cl, args)
+
+	if err == nil {
+		os.Exit(0)
+	}
+
+	switch err.(type) {
+	case cli.UnknownCommandError:
+		fmt.Fprintf(os.Stderr, "Error: %v\n\n", err)
+		fmt.Fprint(os.Stderr, "Commands:\n")
+		printCommands(os.Stderr)
+	case rokka.StatusCodeError:
+		fmt.Fprintf(os.Stderr, "Error: %v\n\n", err)
+		fmt.Fprintf(os.Stderr, "%s\n", cli.PrettyJSON(err.(rokka.StatusCodeError).Body))
```

Comment:
At this point, `err` should already be typehinted without the need to typehint it again. So you should be able to directly write `err.Body`